autoload/airline/extensions/ycm.vim

```diff
@@ -12,8 +17,8 @@ function! airline#extensions#ycm#init(ext)
   call airline#parts#define_function('ycm_warning_count', 'airline#extensions#ycm#get_warning_count')
 endfunction
 
-function! airline#extensions#ycm#get_error_count()
-  if exists(':YcmDiag') && exists("*youcompleteme#GetErrorCount")
+function! airline#extensions#ycm#get_error_count() abort
+  if exists("*youcompleteme#GetErrorCount")
```

Comment:
YcmDiag command was not found in the script of the head family, so I deleted it. (Instead, I added a check using global variables.)

## reference

> https://github.com/ycm-core/YouCompleteMe/blob/master/autoload/youcompleteme.vim#L1118